src/Google/Task/Runner.php

```diff
@@ -269,8 +264,7 @@ public function allowedRetries($code, $errors = array())
       return $this->retryMap[$code];
     }
 
-    if (!empty($errors) && isset($errors[0]['reason']) &&
-        isset($this->retryMap[$errors[0]['reason']])) {
+    if (!empty($errors) && isset($errors[0]['reason'], $this->retryMap[$errors[0]['reason']])) {
```

Comment:
This should be under 80 characters, so it can be wrapped like this:


```php
    if (
        !empty($errors) && 
        isset($errors[0]['reason'], $this->retryMap[$errors[0]['reason']])
    ) {
```